ui/pages/confirmation/templates/add-ethereum-chain.js

```diff
@@ -75,6 +56,48 @@ const INVALID_CHAIN = {
   },
 };
 
+const MISMATCHED_NETWORK_NAME = {
+  id: 'MISMATCHED_NETWORK_NAME',
+  severity: SEVERITIES.WARNING,
+  content: {
+    element: 'span',
+    children: {
+      element: 'MetaMaskTranslation',
+      props: {
+        translationKey: 'mismatchedNetworkName',
+      },
+    },
+  },
+};
+
+const MISMATCHED_NETWORK_SYMBOL = {
+  id: 'MISMATCHED_NETWORK_SYMBOL',
+  severity: SEVERITIES.DANGER,
+  content: {
+    element: 'span',
+    children: {
+      element: 'MetaMaskTranslation',
+      props: {
+        translationKey: 'mismatchedNetworkSymbol',
+      },
+    },
+  },
+};
+
+const MISMATCHED_NETWORK_RPC = {
+  id: 'MISMATCHED_NETWORK_RPC',
+  severity: SEVERITIES.DANGER,
+  content: {
+    element: 'span',
+    children: {
+      element: 'MetaMaskTranslation',
+      props: {
+        translationKey: 'mismatchedRpcUrl',
+      },
+    },
+  },
+};
+
```

Comment:
Here I've disaggregated the error we show when a `wallet_addEthereumChain` call contains parameters we don't expect, so that we show which data doesn't match:
Before:
After
<img width="362" alt="Screen Shot 2022-07-15 at 11 13 57 AM" src="https://user-images.githubusercontent.com/34557516/179264327-bd1c7a4e-8988-408e-8fcb-780d57efac39.png">
:

Reply:
1. Is there a reason you are using two different reds in the warning icon? 
2. If the four fields have a warning, will they stack, preventing the preview from being seen? 
3. Is it possible to add a **bold** in "network name" and "currency symbol"?
4. If the validation is the same across fields, could we simplify that error message? 

`According to our record the network name may not correctly match this chain ID`
`The submitted currency symbol does not match what we expect for this chain ID`

eg. 

`According to our records for this chain ID, these may be wrong: Network Name, Currency Symbol`
`According to what we expect for this chain ID, these may be wrong: Network Name, Currency Symbol`

Reply:
> I was trying to indicate the relative severity of each of these discrepancies.

We may need a pattern for this. I'd recommend to use the same indicator for now.

> Yes this probably makes sense. I will push a follow-up to make this error simpler, using this approach.

Super, thank you.